river/stats/lossy.py

```diff
@@ -108,12 +112,13 @@ def update(self, x: typing.Hashable):
             for key in prune:
                 del self._entries[key]
 
-            self._delta = self._bucket_width + self._delta * self.forgetting
+            self._delta = self._bucket_width + self._delta * self.alpha
 
-    def get(self):
+    def get(self) -> typing.Optional[typing.List[typing.Hashable]]:  # type: ignore
         res = []
         for key in self._entries:
             freq, _ = self._entries[key]
             if freq >= (self.support - self.epsilon) * self._delta:
                 res.append((key, freq))
-        return sorted(res, key=itemgetter(1), reverse=True)
+        if res:
+            return [elem[0] for elem in sorted(res, key=operator.itemgetter(1), reverse=True)]
```

Comment:
I would always return a list, that will simply the return type annotation
```suggestion
        return [elem[0] for elem in sorted(res, key=operator.itemgetter(1), reverse=True)]
```

Reply:
I had to do that because of a failing test :)

I think it was the test that ensures `repr` is valid when the stat was not yet updated.

Reply:
When returning `[]` the test fails so I made it return `None` when no data was observed. What are your thoughts about that @MaxHalford? 

Keep `None` or should we update the test?

Reply:
Mmm I see. Let's go with `None`, it's fine :)